### ISSUE TITLE
Travel Advice Styling

### ISSUE THREAD

**Issue by brunobowden** [WorldHealthOrganization/app#1547]
Change the styling on the Travel Advice page. See the annotated screenshot below. In particular the title and body of all the non-banner elements use the same font style (Black and Bold), so it means that you can't distinguish between the title and body, nor can you scan the titles only to get a summary of the advice. The new styling should be applied to all non-banner items.

- [x] Searched the [existing issues](https://github.com/WorldHealthOrganization/app/issues) to ensure you are not creating a duplicate.
- [x] Followed the [Contributor Guidelines](https://github.com/WorldHealthOrganization/app/blob/master/docs/CONTRIBUTING.md).
- [x] Issue has descriptive title focused on the feature or problem from the user's perspective

![56315](https://user-images.githubusercontent.com/679339/95003024-9bf63c00-058f-11eb-83b3-5afb4e184d0f.jpg)

**Comment by brunobowden** [comment]
@theswerd @britannio - it would be a great help if either of you could send a PR for this.

**Comment by brunobowden** [comment]
Turns out that this happened due to enabling "Bold Text" on my iOS settings. Without that, it looks fine. I think the remaining issues aren't important enough to worry about.
